features/activity/cancel_try_cancel/feature.ts

```diff
@@ -0,0 +1,48 @@
+import { Context } from '@temporalio/activity';
+import { WorkflowClient } from '@temporalio/client';
+import { CancelledFailure } from '@temporalio/common';
+import { Feature } from '@temporalio/harness';
+import { workflow, activityResultSignal } from './feature.workflow';
+
+export const feature = new Feature({
+  workflow,
+  activities: {
+    cancellableActivity,
+  },
+  execute: async (runner) => {
+    client = runner.client;
+    return await runner.executeSingleParameterlessWorkflow();
+  },
+});
+
+let client: WorkflowClient | undefined;
```

Comment:
Not loving this global variable.
Would have been much nicer to have a way to instantiate an activity class (or closure) or even inject this in an activity interceptor.

Reply:
This feature is not yet released in Python